src/sysinfo/nvapi_adapter.cpp

```diff
@@ -130,9 +130,14 @@ namespace dxvk {
 
             nvmlDevice_t nvmlDevice{};
             auto result = m_nvml.DeviceGetHandleByPciBusId_v2(pciId, &nvmlDevice);
-            if (result == NVML_SUCCESS)
+            if (result == NVML_SUCCESS) {
                 m_nvmlDevice = nvmlDevice;
-            else
+
+                nvmlMemory_v2_t memory{};
```

Comment:
You should set the `version` field to `nvmlMemory_v2` here, otherwise the call below will most likely fail.

Reply:
I'm setting the version here https://github.com/jp7677/dxvk-nvapi/blob/fc0b9a7ab8e9763359ad8d38a6fd4cbbd24d4e3b/src/sysinfo/nvml.cpp#L81